src/sphinxtheme/plone/plone_basic/layout.html

```diff
@@ -102,6 +102,7 @@
   <link rel="icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" type="image/x-icon">
   <link rel="shortcut icon" type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
   <link type="image/x-icon" href="{{ pathto('_static/' + theme_favicon, 1) }}" />
+  
```

Comment:
A blank line is also not needed. It is a good practice to restore code to its original when removing proposed changes to minimize the diff.

```suggestion
```